pkgs/development/python-modules/django-maintenance-mode/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchPypi, buildPythonPackage,
+  mock, django }:
+buildPythonPackage rec {
+  pname = "django-maintenance-mode";
+  version = "0.13.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "16yfw85r6my14vbrf9g2hj5p7ivgjvrd16s9prsjzqj9k8qikch2";
+  };
+
+  doCheck = false;
+
+  buildInputs = [ mock ];
```

Comment:
Why is this needed? It works (and the tests pass) for me without it (python 2.7 & 3.7).

Reply:
A good question indeed. I'll remove it and RTFM.